### ISSUE TITLE
python 'line' built in function crashes when called

### ISSUE THREAD

**Issue by jezv** [nesbox/TIC-80#2316]
version 1.1.2736 (59ab359)
windows 11

repro steps:

1.  .\tic80.exe --fs=./test --scale=6
2. new python
3. ESC to enter code editor
4. below the hello world print, add the following line call
```
 print("HELLO WORLD!",84,84)
 line(84,84,91,91,1)
```
 
 at runtime you I get the following error
```
>error while running TIC

>Traceback (most recent call last):
  File "<c-bound>", line 32
    line(84,84,91,91,1)
  File "<c-bound>", line 1
    def line(x0, y0, x1, y1, color) : 
_line(int(x0), int(y0), int(x1), 
int(y1), int(color))
TypeError: expected 'float', but got 
'int'
```

```line(84.0,84.0,91.0,91.0,1.0)```

this gives me the same result

**Comment by blueloveTH** [comment]
I think this was solved by https://github.com/nesbox/TIC-80/pull/2315

**Comment by nesbox** [comment]
fixed with #2315
